common/djangoapps/third_party_auth/signals/handlers.py

```diff
@@ -0,0 +1,61 @@
+"""
+Signal handlers for third_party_auth app.
+"""
+
+from django.db.models.signals import post_save
+from django.dispatch import receiver
+from edx_django_utils.monitoring import set_custom_attribute
+
+
+from common.djangoapps.third_party_auth.models import SAMLConfiguration, SAMLProviderConfig
+
+
+from common.djangoapps.third_party_auth.toggles import ENABLE_SAML_CONFIG_SIGNAL_HANDLERS
+
+
+@receiver(post_save, sender=SAMLConfiguration)
+def update_saml_provider_configs_on_configuration_change(sender, instance, created, **kwargs):
+    """
+    NOTE: This behavior is controlled by the ENABLE_SAML_CONFIG_SIGNAL_HANDLERS toggle.
+
```

Comment:
Two reasons to remove this:
1. The toggle is for temporary rollout, and will probably only affect us, so I'd leave this out. It is clear from the code.
2. The first line of a docstring should be the one-line summary description of the function. 
```suggestion
    NOTE: This behavior is controlled by the ENABLE_SAML_CONFIG_SIGNAL_HANDLERS toggle.

```